aspnetcore/fundamentals/metapackage.md

```diff
@@ -34,6 +33,18 @@ The following *.csproj* file references the `Microsoft.AspNetCore.All` metapacka
 
 [!code-xml[](metapackage/samples/Metapackage.All.Example.csproj?highlight=6)]
 
+The preceding markup represents a typical ASP.NET Core 2.0 template. It doesn't specify a version number for the `Microsoft.AspNetCore.All` package reference. When the version isn't specified, an [implicit](https://github.com/dotnet/core/blob/master/release-notes/1.0/sdk/1.0-rc3-implicit-package-refs.md) version is specified by the SDK (`Microsoft.NET.Sdk.Web`). We recommend relying on the implicit version specified by the SDK and not explicitly setting the version number on the package reference. If you have questions about this approach, leave a GitHub comment at the [Discussion for the Microsoft.AspNetCore.App implicit version](https://github.com/aspnet/Docs/issues/6430).
+
+The implicit version is set to `major.minor.0` for portable apps. The shared framework roll-forward mechanism runs the app on the latest compatible version among the installed shared frameworks. To guarantee the same version is used in development, test, and production, ensure the same version of the shared framework is installed in all environments. For self-contained apps, the implicit version number is set to the `major.minor.patch` of the shared framework bundled in the installed SDK.
+
+Specifying a version number on the `Microsoft.AspNetCore.All` package reference does **not** guarantee that version of the shared framework is chosen. For example, suppose version "2.0.0" is specified, but "2.0.1" is installed. In that case, the app will use "2.0.1". Although not recommended, you can disable roll forward (patch and/or minor). For more information regarding dotnet host roll-forward and how to configure its behavior, see [dotnet host roll forward](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/roll-forward-on-no-candidate-fx.md).
```

Comment:
Let's use 2.1.1 and 2.1.3 as examples instead since 2.1 is the first time we have this behaviour. This would also bring the docs to parity with what's in the metapackage-app.md file.